### DISCUSSION TITLE
Manually disconnecting and reconnecting chromedriver

### DISCUSSION THREAD

**Discussion by LimitedTime19** [seleniumbase/SeleniumBase#2245] [Q&A] [answered]
Hi, I was wondering if you could help and teach me how to manually disconnecting and reconnecting chromedriver whenever we need. I understand things like driver.get or driver clicking commands automatically does, but I want to be able to try it without driver commands myself and decide when I want to hide chromedriver when needed.

**Comment by mdmintz** [comment, marked as answer]
These are the methods you need for specialized UC Mode usage:

```python
driver.uc_open(url)
driver.uc_open_with_tab(url)
driver.uc_open_with_reconnect(url, reconnect_time=None)
driver.uc_click(selector)
```

All the details can be found in this video: https://www.youtube.com/watch?v=5dMFI3e85ig
